cmd/crc/cmd/root.go

```diff
@@ -146,5 +146,10 @@ func newMachine() machine.Client {
 }
 
 func newMachineWithConfig(config crcConfig.Storage) machine.Client {
-	return machine.NewClient(constants.DefaultName, isDebugLog(), network.ParseMode(config.Get(cmdConfig.NetworkMode).AsString()))
+	return machine.NewClient(
+		constants.DefaultName,
+		isDebugLog(),
+		network.ParseMode(config.Get(cmdConfig.NetworkMode).AsString()),
+		config.Get(cmdConfig.EnableOpenShiftMonitoring).AsBool(),
```

Comment:
Random thought: 4 arguments is starting to be too much from a readability perspective, when a 5th one is considered, it would be nice to refactor this... For example by introducing an intermediate `ClientConfig` type, or by removing some args ,and adding some setters, `if isDebugLog() client.SetDebug(true)` and so on.

Reply:
That sounds like a citation from Clean Code! :)

Reply:
Never read it, so definitely not, but could be some remnants from Code Complete which I read a long time ago :) They probably are similar books.